app/external-api/src/main/java/com/bombombom/devs/external/study/service/StudyService.java

```diff
@@ -71,18 +71,36 @@ public List<Round> findRoundsHaveToStart() {
 
     @Transactional(readOnly = true)
     public StudyDetailsResult findStudyDetails(Long studyId) {
-        Study study = studyRepository.findById(studyId)
+        Study study = studyRepository.findWithDifficultiesAndLeaderAndBookById(studyId)
             .orElseThrow(() -> new NotFoundException(ErrorCode.STUDY_NOT_FOUND));
 
         Round currentRound = roundRepository.findRoundByStudyIdAndBetweenStartDateAndEndDateOrIdx(
                 studyId, study.getWeeks() - 1, clock.today())
             .orElseThrow(() -> new NotFoundException(ErrorCode.ROUND_NOT_FOUND));
-        return StudyDetailsResult.fromResult(study, findStudyProgress(study, currentRound));
+        return new StudyDetailsResult(StudyResult.fromEntity(study),
+            findStudyProgress(study, currentRound));
+    }
+
+    @Transactional
+    public StudyResult configure(Long userId,
+        Long studyId, ConfigureStudyCommand command) {
```

Comment:
>   Successful PATCH response to existing text file:
 HTTP/1.1 204 No Content
   Content-Location: /file.txt
   ETag: "e0023aa4f"
  The 204 response code is used because the response does not carry a
   message body (which a response with the 200 code would have).  Note
   that other success codes could be used as well.

[RFC 5789 PATCH Method for HTTP](https://www.rfc-editor.org/rfc/rfc5789#section-2.1) 
있는 내용입니다. 
body가 없는 204 코드로 응답하거나 body (리소스에 대한 표현)가 있는 200코드로 응답하거나인 것 같은데

[patch 메소드의 응답?](https://stackoverflow.com/questions/54666244/what-should-response-of-rest-full-api-for-patch-http-method)
두 경우를 어떻게 구분하는지에 대한 명확한 지침은 없어서 검색해보니 일반적으로는 patch 요청을 한 클라이언트가 변경된 리소스의 상태를 추론가능할때는 204를 쓰는 것 같습니다. 200은 반대로 추론이 불가능할때에 변경된 리소스 상태를 body로 응답한다고 하네요.

제 경우에는 추론이 가능하니 204에 body가 없는 응답을 보내는게 맞는 것 같습니다. 질문 감사합니다!